pkg/packages.go

```diff
@@ -41,12 +40,8 @@ func Install(ctx context.Context, m spec.JsonnetFile, dir string) (lock *spec.Js
 		if err != nil {
 			return nil, errors.Wrap(err, "failed to create general tmp dir")
 		}
-		tmpDir, err := ioutil.TempDir(tmp, fmt.Sprintf("jsonnetpkg-%s-%s", dep.Name, dep.Version))
-		if err != nil {
-			return nil, errors.Wrap(err, "failed to create tmp dir")
-		}
-		defer os.RemoveAll(tmpDir)
 
+		tmpDir := filepath.Join(tmp, dep.Name)
```

Comment:
We could consider putting this whole directory outside of the repo (in `~/.jsonnet.cache` or something)?

Reply:
How would that result in caching different versions of the same packages? Do we simply copy those versions over?

Reply:
a global cache would be neat, but needs some more thought (jb actually started with a global cache and I decided to skip it for the beginning as the complexity quickly rose, it might be time to revisit it though)

Reply:
> How would that result in caching different versions of the same packages? 

The code now _always_ checks out the version before the copy, so it shouldn't copy over the wrong version.